src/common/components/Translator.tsx

```diff
@@ -1195,29 +1195,31 @@ function InnerTranslator(props: IInnerTranslatorProps) {
         translatedStopSpeakRef.current = stopSpeak
     }
 
+    const handleReadSelectedWordsFromInputElementsAction = useCallback(async () => {
+        if (isSpeakingSelectedWordsFromInputElements.current === true) {
+            selectedWordsFromInputElementsStopSpeakRef.current()
+            isSpeakingSelectedWordsFromInputElements.current = false
+            return
+        }
+        isSpeakingSelectedWordsFromInputElements.current = true
```

Comment:
`isSpeakingSelectedWordsFromInputElements` uses `useRef` instead of `useState` because we do not want `isSpeakingSelectedWordsFromInputElements` to cause `useCallback` to return a new function value.